implementation/src/main/java/io/smallrye/mutiny/Uni.java

```diff
@@ -690,6 +690,30 @@ static UniCombine combine() {
         return onItem().transform(ignored -> null);
     }
 
+    /**
+     * When this {@link Uni} emits {@code null}, replace with the value provided by the given {@link Supplier}.
+     *
+     * This is a shortcut for {@code uni.onItem().ifNull().continueWith(supplier)}
+     *
+     * @param supplier the supplier
+     * @return the new {@link Uni}
+     */
+    default Uni<T> replaceIfNullWith(Supplier<T> supplier) {
```

Comment:
Should we add it to the shortcut list we have in the doc?

Reply:
Right!

Reply:
Oh there are a bunch missing, adding them